mdoc/src/main/scala/mdoc/internal/io/IO.scala

```diff
@@ -42,28 +43,30 @@ object IO {
 
   def foreachInputFile(settings: Settings)(fn: InputFile => Unit): Unit = {
     implicit val cwd = settings.cwd
-    val root = settings.in.toNIO
-    val visitor = new SimpleFileVisitor[Path] {
-      override def visitFile(
-          file: Path,
-          attrs: BasicFileAttributes
-      ): FileVisitResult = {
-        settings.toInputFile(AbsolutePath(file)) match {
-          case Some(inputFile) =>
-            fn(inputFile)
-          case None =>
-            () // excluded
+    def handle(file: AbsolutePath): Unit =
+      settings.toInputFile(file).foreach(fn)
+    settings.in.foreach { root =>
+      val visitor = new SimpleFileVisitor[Path] {
+        override def visitFile(
+            file: Path,
+            attrs: BasicFileAttributes
+        ): FileVisitResult = {
+          handle(AbsolutePath(file))
+          FileVisitResult.CONTINUE
         }
-        FileVisitResult.CONTINUE
-      }
 
-      override def preVisitDirectory(dir: Path, attrs: BasicFileAttributes): FileVisitResult = {
-        val relpath = RelativePath(root.relativize(dir))
-        if (settings.isExplicitlyExcluded(relpath)) FileVisitResult.SKIP_SUBTREE
-        else FileVisitResult.CONTINUE
+        override def preVisitDirectory(dir: Path, attrs: BasicFileAttributes): FileVisitResult = {
+          val relpath = RelativePath(root.toNIO.relativize(dir))
+          if (settings.isExplicitlyExcluded(relpath)) FileVisitResult.SKIP_SUBTREE
```

Comment:
Super nitpicky, but everywhere else it seems we are doing `if` `else` statements with brackets except for here. We may want to be consistent.

Reply:
I think this is how it was formatted before. Either way, fixed